2018-11-07 16:06:29,990 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-07 16:07:50,239 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-08 10:20:07,696 INFO - -------------------------------------------------------------------------------------
2018-11-08 10:20:07,696 INFO - Logged in by: consys
2018-11-08 10:20:21,628 INFO - Employee Details page viewed by : consys
2018-11-09 08:31:00,012 ERROR - Login attempt failed
2018-11-09 08:31:00,012 ERROR - Failed to login for 2118434
2018-11-09 08:31:19,647 INFO - -------------------------------------------------------------------------------------
2018-11-09 08:31:19,647 INFO - Logged in by: convmis
2018-11-09 08:31:36,826 INFO - Leave applied for convmis on 07/11/2018 by: convmis
2018-11-09 08:31:51,628 INFO - Employee Details page viewed by : convmis
2018-11-09 08:44:20,438 INFO - Logged out by user..
2018-11-09 08:44:20,438 INFO - -------------------------------------------------------------------------------------
2018-11-09 09:36:22,610 ERROR - Login attempt failed
2018-11-09 09:36:22,610 ERROR - Failed to login for conmgur
2018-11-09 09:36:35,917 INFO - -------------------------------------------------------------------------------------
2018-11-09 09:36:35,917 INFO - Logged in by: conmgur
2018-11-09 09:36:51,965 INFO - Leave applied for conmgur on 07/11/2018 by: conmgur
2018-11-09 11:07:34,713 INFO - -------------------------------------------------------------------------------------
2018-11-09 11:07:34,713 INFO - Logged in by: conplo
2018-11-09 11:07:57,042 INFO - Leave applied for conplo on 07/11/2018 by: conplo
2018-11-12 12:30:28,234 INFO - -------------------------------------------------------------------------------------
2018-11-12 12:30:28,265 INFO - Logged in by: conddas
2018-11-12 12:32:11,236 INFO - Employee Details page viewed by : conddas
2018-11-12 12:33:01,092 INFO - Logged out by user..
2018-11-12 12:33:01,092 INFO - -------------------------------------------------------------------------------------
2018-11-15 14:59:46,724 INFO - -------------------------------------------------------------------------------------
2018-11-15 14:59:46,740 INFO - Logged in by: conddas
2018-11-15 14:59:54,644 INFO - Employee Details page viewed by : conddas
2018-11-15 15:00:26,800 INFO - Logged out by user..
2018-11-15 15:00:26,800 INFO - -------------------------------------------------------------------------------------
2018-11-15 15:14:31,363 INFO - -------------------------------------------------------------------------------------
2018-11-15 15:14:31,363 INFO - Logged in by: conpnag
2018-11-15 15:15:17,258 INFO - Leave applied for conpnag on 05/11/2018 by: conpnag
2018-11-15 15:15:26,001 INFO - Leave applied for conpnag on 09/11/2018 by: conpnag
2018-11-15 15:16:59,327 INFO - Employee Details page viewed by : conpnag
2018-11-15 15:25:00,303 INFO - Logged out by user..
2018-11-15 15:25:00,303 INFO - -------------------------------------------------------------------------------------
2018-11-15 17:37:43,935 INFO - -------------------------------------------------------------------------------------
2018-11-15 17:37:43,935 INFO - Logged in by: conpnag
2018-11-15 17:37:50,596 INFO - Leave applied for conpnag on 07/11/2018 by: conpnag
2018-11-15 17:38:53,709 INFO - Logged out by user..
2018-11-15 17:38:53,709 INFO - -------------------------------------------------------------------------------------
2018-11-16 09:13:13,753 INFO - -------------------------------------------------------------------------------------
2018-11-16 09:13:13,753 INFO - Logged in by: consys
2018-11-16 10:21:50,626 INFO - -------------------------------------------------------------------------------------
2018-11-16 10:21:50,626 INFO - Logged in by: consqua
2018-11-16 10:22:39,176 INFO - -------------------------------------------------------------------------------------
2018-11-16 10:22:39,176 INFO - Logged in by: consqua
2018-11-16 10:23:27,667 INFO - -------------------------------------------------------------------------------------
2018-11-16 10:23:27,682 INFO - Logged in by: consqua
2018-11-16 10:24:16,158 INFO - -------------------------------------------------------------------------------------
2018-11-16 10:24:16,158 INFO - Logged in by: consqua
2018-11-16 10:25:04,617 INFO - -------------------------------------------------------------------------------------
2018-11-16 10:25:04,617 INFO - Logged in by: consqua
2018-11-16 11:06:38,544 INFO - Leave applied for consqua on 09/11/2018 by: consqua
2018-11-16 11:06:53,278 INFO - Leave applied for consqua on 12/11/2018 by: consqua
2018-11-16 11:06:59,432 INFO - Leave applied for consqua on 13/11/2018 by: consqua
2018-11-16 11:07:06,034 INFO - Logged out by user..
2018-11-16 11:07:06,034 INFO - -------------------------------------------------------------------------------------
2018-11-16 11:20:07,327 INFO - -------------------------------------------------------------------------------------
2018-11-16 11:20:07,327 INFO - Logged in by: conahn
2018-11-16 11:20:55,980 INFO - -------------------------------------------------------------------------------------
2018-11-16 11:20:55,980 INFO - Logged in by: conahn
2018-11-16 11:21:44,524 INFO - -------------------------------------------------------------------------------------
2018-11-16 11:21:44,524 INFO - Logged in by: conahn
2018-11-16 11:22:33,069 INFO - -------------------------------------------------------------------------------------
2018-11-16 11:22:33,069 INFO - Logged in by: conahn
2018-11-16 11:24:25,237 INFO - Leave applied for conahn on 19/11/2018 by: conahn
2018-11-19 11:39:07,814 INFO - -------------------------------------------------------------------------------------
2018-11-19 11:39:07,814 INFO - Logged in by: conddas
2018-11-19 11:39:33,298 INFO - Employee Details page viewed by : conddas
2018-11-19 11:39:39,588 INFO - Logged out by user..
2018-11-19 11:39:39,588 INFO - -------------------------------------------------------------------------------------
2018-11-19 15:58:13,969 INFO - -------------------------------------------------------------------------------------
2018-11-19 15:58:13,969 INFO - Logged in by: conravh
2018-11-19 15:58:47,973 INFO - Leave applied for conravh on 07/11/2018 by: conravh
2018-11-19 16:40:10,979 INFO - Logged out by user..
2018-11-19 16:40:10,979 INFO - -------------------------------------------------------------------------------------
2018-11-19 16:42:09,503 INFO - -------------------------------------------------------------------------------------
2018-11-19 16:42:09,503 INFO - Logged in by: convsiv
2018-11-19 16:42:48,454 INFO - Leave applied for convsiv on 02/11/2018 by: convsiv
2018-11-19 16:43:01,048 INFO - Employee Details page viewed by : convsiv
2018-11-19 16:43:15,088 INFO - Logged out by user..
2018-11-19 16:43:15,088 INFO - -------------------------------------------------------------------------------------
2018-11-19 17:12:12,328 ERROR - Login attempt failed
2018-11-19 17:12:12,328 ERROR - Failed to login for 2350324
2018-11-19 17:12:32,886 ERROR - Login attempt failed
2018-11-19 17:12:32,886 ERROR - Failed to login for 2350324
2018-11-19 17:13:12,788 ERROR - Login attempt failed
2018-11-19 17:13:12,788 ERROR - Failed to login for 2350324
2018-11-19 17:13:49,074 ERROR - Login attempt failed
2018-11-19 17:13:49,074 ERROR - Failed to login for 235034
2018-11-19 17:14:11,678 ERROR - Login attempt failed
2018-11-19 17:14:11,678 ERROR - Failed to login for 2350324
2018-11-19 17:17:55,873 ERROR - Login attempt failed
2018-11-19 17:17:55,873 ERROR - Failed to login for srinivasan.s01@mphasis.com
2018-11-19 17:37:24,118 INFO - -------------------------------------------------------------------------------------
2018-11-19 17:37:24,118 INFO - Logged in by: convbha
2018-11-19 17:37:52,387 INFO - Leave applied for convbha on 09/11/2018 by: convbha
2018-11-19 17:38:18,652 INFO - Logged out by user..
2018-11-19 17:38:18,652 INFO - -------------------------------------------------------------------------------------
2018-11-19 17:49:57,446 INFO - -------------------------------------------------------------------------------------
2018-11-19 17:49:57,446 INFO - Logged in by: congeth
2018-11-19 17:50:24,261 INFO - Leave applied for congeth on 02/11/2018 by: congeth
2018-11-19 17:50:35,056 INFO - Leave applied for congeth on 07/11/2018 by: congeth
2018-11-19 17:50:43,635 INFO - Leave applied for congeth on 09/11/2018 by: congeth
2018-11-19 17:50:55,213 INFO - Employee Details page viewed by : congeth
2018-11-19 17:51:23,299 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-19 17:51:36,824 INFO - Leave applied for conrha on 02/11/2018 by: conrha
2018-11-19 17:51:50,902 INFO - Leave applied for conrha on 13/11/2018 by: conrha
2018-11-19 17:52:28,598 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-19 17:52:34,700 INFO - Leave applied for conshshi on 05/11/2018 by: conshshi
2018-11-19 17:52:43,584 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-19 17:52:49,980 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-19 17:55:29,864 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-19 17:55:45,931 INFO - Leave applied for conshshi on 07/11/2018 by: conshshi
2018-11-19 17:55:52,963 INFO - Leave applied for conshshi on 09/11/2018 by: conshshi
2018-11-19 17:56:15,731 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-19 17:56:33,897 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-19 17:56:41,293 INFO - Leave applied for conavk on 05/11/2018 by: conavk
2018-11-19 17:56:53,768 INFO - Leave applied for conavk on 07/11/2018 by: conavk
2018-11-19 17:57:07,894 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-19 17:57:25,982 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-19 17:57:30,680 INFO - Leave applied for consasou on 07/11/2018 by: consasou
2018-11-19 17:57:35,594 INFO - Leave applied for consasou on 09/11/2018 by: consasou
2018-11-19 17:57:52,936 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-19 17:57:59,544 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-19 17:58:09,310 INFO - Leave applied for conddas on 09/11/2018 by: conddas
2018-11-20 09:53:34,094 INFO - -------------------------------------------------------------------------------------
2018-11-20 09:53:34,094 INFO - Logged in by: conspatel
2018-11-20 09:53:42,584 INFO - -------------------------------------------------------------------------------------
2018-11-20 09:53:42,584 INFO - Logged in by: conspatel
2018-11-20 09:54:49,813 INFO - Leave applied for conspatel on 15/11/2018 by: conspatel
2018-11-20 09:55:13,061 INFO - Logged out by user..
2018-11-20 09:55:13,061 INFO - -------------------------------------------------------------------------------------
2018-11-20 11:07:29,607 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-20 11:07:39,309 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-20 13:24:43,978 INFO - -------------------------------------------------------------------------------------
2018-11-20 13:24:43,978 INFO - Logged in by: conddas
2018-11-20 13:25:03,277 INFO - Employee Details page viewed by : conddas
2018-11-20 13:30:25,736 INFO - Logged out by user..
2018-11-20 13:30:25,736 INFO - -------------------------------------------------------------------------------------
2018-11-20 14:05:09,526 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-20 14:06:59,271 INFO - Logged out by user..
2018-11-20 14:06:59,271 INFO - -------------------------------------------------------------------------------------
2018-11-21 14:17:13,003 INFO - -------------------------------------------------------------------------------------
2018-11-21 14:17:13,003 INFO - Logged in by: conshal
2018-11-21 14:17:31,472 INFO - Leave applied for conshal on 20/11/2018 by: conshal
2018-11-22 10:29:50,638 INFO - -------------------------------------------------------------------------------------
2018-11-22 10:29:50,638 INFO - Logged in by: congeth
2018-11-22 10:29:54,408 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-22 10:30:06,594 INFO - Leave applied for consqua on 21/11/2018 by: consqua
2018-11-22 10:30:12,100 INFO - Leave applied for consqua on 22/11/2018 by: consqua
2018-11-22 10:30:17,815 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-22 11:17:41,570 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-22 13:34:29,838 INFO - Logged out by user..
2018-11-22 13:34:29,838 INFO - -------------------------------------------------------------------------------------
2018-11-22 15:00:12,099 INFO - -------------------------------------------------------------------------------------
2018-11-22 15:00:12,114 INFO - Logged in by: conank
2018-11-22 15:00:27,790 INFO - Leave applied for conank on 23/11/2018 by: conank
2018-11-22 15:00:40,987 INFO - Logged out by user..
2018-11-22 15:00:40,987 INFO - -------------------------------------------------------------------------------------
2018-11-22 15:30:24,223 INFO - -------------------------------------------------------------------------------------
2018-11-22 15:30:24,223 INFO - Logged in by: contmg
2018-11-22 15:31:03,213 INFO - Leave applied for contmg on 20/11/2018 by: contmg
2018-11-22 15:32:31,774 INFO - Logged out by user..
2018-11-22 15:32:31,774 INFO - -------------------------------------------------------------------------------------
2018-11-26 08:46:23,319 INFO - -------------------------------------------------------------------------------------
2018-11-26 08:46:23,319 INFO - Logged in by: convmis
2018-11-26 08:46:50,993 INFO - -------------------------------------------------------------------------------------
2018-11-26 08:46:50,993 INFO - Logged in by: convmis
2018-11-26 08:49:40,101 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-26 08:50:31,900 INFO - Leave applied for conmgur on 16/11/2018 by: conmgur
2018-11-26 08:50:36,750 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-26 08:50:36,765 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-26 08:52:20,514 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-26 08:52:24,518 INFO - Leave applied for conrank on 26/11/2018 by: conrank
2018-11-26 08:52:28,247 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-26 08:52:28,247 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-26 08:54:24,235 INFO - Leave applied for convmoh on 12/11/2018 by: convmoh
2018-11-26 08:54:28,509 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-26 08:54:28,509 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-26 08:56:32,144 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-26 08:58:05,910 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-26 08:58:24,638 INFO - Leave applied for connkh on 07/11/2018 by: connkh
2018-11-26 08:58:31,180 INFO - Leave applied for connkh on 19/11/2018 by: connkh
2018-11-26 08:58:35,191 INFO - Leave applied for connkh on 20/11/2018 by: connkh
2018-11-26 08:58:39,396 INFO - Leave applied for connkh on 21/11/2018 by: connkh
2018-11-26 08:58:43,213 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-26 08:58:43,223 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-26 08:59:34,563 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-26 08:59:41,216 INFO - Leave applied for conntuk on 09/11/2018 by: conntuk
2018-11-26 08:59:43,458 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-26 08:59:47,681 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-26 09:00:29,750 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-26 09:43:02,909 INFO - -------------------------------------------------------------------------------------
2018-11-26 09:43:02,909 INFO - Logged in by: conravh
2018-11-26 09:43:07,260 INFO - -------------------------------------------------------------------------------------
2018-11-26 09:43:07,260 INFO - Logged in by: conravh
2018-11-26 09:58:34,756 INFO - Employee Details page viewed by : conravh
2018-11-26 09:58:54,768 INFO - Leave applied for conravh on 28/11/2018 by: conravh
2018-11-26 09:59:07,825 INFO - Employee Details page viewed by : conravh
2018-11-26 11:03:27,299 INFO - -------------------------------------------------------------------------------------
2018-11-26 11:03:27,299 INFO - Logged in by: congeth
2018-11-26 11:03:33,943 INFO - -------------------------------------------------------------------------------------
2018-11-26 11:03:33,943 INFO - Logged in by: congeth
2018-11-26 11:03:40,656 INFO - -------------------------------------------------------------------------------------
2018-11-26 11:03:40,656 INFO - Logged in by: congeth
2018-11-26 11:04:09,809 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-26 11:04:14,952 INFO - Leave applied for consqua on 23/11/2018 by: consqua
2018-11-26 11:04:22,303 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-26 12:22:45,433 INFO - Logged out by user..
2018-11-26 12:22:45,433 INFO - -------------------------------------------------------------------------------------
2018-11-26 16:32:30,872 ERROR - Login attempt failed
2018-11-26 16:32:30,872 ERROR - Failed to login for 2350324
2018-11-26 16:32:47,926 ERROR - Login attempt failed
2018-11-26 16:32:47,926 ERROR - Failed to login for 2350324
2018-11-26 16:34:05,874 INFO - -------------------------------------------------------------------------------------
2018-11-26 16:34:05,874 INFO - Logged in by: conssud
2018-11-26 16:34:28,734 INFO - Logged out by user..
2018-11-26 16:34:28,734 INFO - -------------------------------------------------------------------------------------
2018-11-26 17:52:26,805 INFO - -------------------------------------------------------------------------------------
2018-11-26 17:52:26,805 INFO - Logged in by: consbr
2018-11-26 17:53:49,716 INFO - Leave applied for consbr on 23/11/2018 by: consbr
2018-11-28 11:59:24,314 INFO - -------------------------------------------------------------------------------------
2018-11-28 11:59:24,329 INFO - Logged in by: congeth
2018-11-28 11:59:31,336 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-28 11:59:48,568 INFO - Leave applied for conrha on 27/11/2018 by: conrha
2018-11-28 13:13:01,689 INFO - -------------------------------------------------------------------------------------
2018-11-28 13:13:01,689 INFO - Logged in by: conpkr
2018-11-28 13:13:08,146 INFO - -------------------------------------------------------------------------------------
2018-11-28 13:13:08,146 INFO - Logged in by: conpkr
2018-11-28 13:13:39,780 INFO - Leave applied for conpkr on 27/11/2018 by: conpkr
2018-11-28 15:11:05,255 INFO - Logged out by user..
2018-11-28 15:11:05,255 INFO - -------------------------------------------------------------------------------------
2018-11-29 10:04:59,668 INFO - -------------------------------------------------------------------------------------
2018-11-29 10:04:59,668 INFO - Logged in by: conplo
2018-11-29 10:05:16,329 INFO - Leave applied for conplo on 28/11/2018 by: conplo
2018-11-29 11:33:50,797 ERROR - Login attempt failed
2018-11-29 11:33:50,797 ERROR - Failed to login for conpnag
2018-11-29 11:34:06,784 INFO - -------------------------------------------------------------------------------------
2018-11-29 11:34:06,784 INFO - Logged in by: convsiv
2018-11-29 11:44:03,610 INFO - -------------------------------------------------------------------------------------
2018-11-29 11:44:03,610 INFO - Logged in by: conpnag
2018-11-29 11:44:20,445 INFO - Employee Details page viewed by : conpnag
2018-11-29 11:44:24,094 INFO - Logged out by user..
2018-11-29 11:44:24,094 INFO - -------------------------------------------------------------------------------------
2018-11-29 13:54:42,094 INFO - Leave applied for convsiv on 24/12/2018 by: convsiv
2018-11-29 13:54:50,656 INFO - Leave applied for convsiv on 25/12/2018 by: convsiv
2018-11-29 13:55:02,416 INFO - Logged out by user..
2018-11-29 13:55:02,416 INFO - -------------------------------------------------------------------------------------
2018-11-29 17:54:51,560 INFO - -------------------------------------------------------------------------------------
2018-11-29 17:54:51,560 INFO - Logged in by: conddas
2018-11-29 18:01:33,889 INFO - Logged out by user..
2018-11-29 18:01:33,889 INFO - -------------------------------------------------------------------------------------
2018-11-30 09:57:07,319 INFO - -------------------------------------------------------------------------------------
2018-11-30 09:57:07,319 INFO - Logged in by: conspatel
2018-11-30 09:57:34,152 INFO - Leave applied for conspatel on 28/11/2018 by: conspatel
2018-11-30 10:01:26,610 INFO - Logged out by user..
2018-11-30 10:01:26,610 INFO - -------------------------------------------------------------------------------------
2018-11-30 10:22:02,289 INFO - -------------------------------------------------------------------------------------
2018-11-30 10:22:02,289 INFO - Logged in by: conpkr
2018-11-30 10:22:29,833 INFO - -------------------------------------------------------------------------------------
2018-11-30 10:22:29,833 INFO - Logged in by: conpkr
2018-11-30 10:22:57,649 INFO - -------------------------------------------------------------------------------------
2018-11-30 10:22:57,649 INFO - Logged in by: conpkr
2018-11-30 10:23:56,176 INFO - -------------------------------------------------------------------------------------
2018-11-30 10:23:56,176 INFO - Logged in by: conpkr
2018-11-30 10:24:06,179 INFO - Leave applied for conpkr on 29/11/2018 by: conpkr
2018-11-30 11:34:07,993 INFO - -------------------------------------------------------------------------------------
2018-11-30 11:34:07,993 INFO - Logged in by: conhgop
2018-11-30 11:35:22,579 INFO - Leave applied for conhgop on 14/11/2018 by: conhgop
2018-11-30 11:35:58,223 INFO - Leave applied for conhgop on 26/11/2018 by: conhgop
2018-11-30 11:36:25,160 INFO - Logged out by user..
2018-11-30 11:36:25,160 INFO - -------------------------------------------------------------------------------------
2018-11-30 13:58:47,046 INFO - -------------------------------------------------------------------------------------
2018-11-30 13:58:47,046 INFO - Logged in by: congeth
2018-11-30 13:59:02,830 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-11-30 13:59:29,120 INFO - Leave applied for conavk on 30/11/2018 by: conavk
2018-12-03 09:05:17,435 INFO - -------------------------------------------------------------------------------------
2018-12-03 09:05:17,435 INFO - Logged in by: conpnag
2018-12-03 09:05:36,534 INFO - Leave applied for conpnag on 30/11/2018 by: conpnag
2018-12-03 09:06:11,137 INFO - Logged out by user..
2018-12-03 09:06:11,137 INFO - -------------------------------------------------------------------------------------
2018-12-03 15:03:42,492 INFO - -------------------------------------------------------------------------------------
2018-12-03 15:03:42,492 INFO - Logged in by: conddas
2018-12-03 15:04:16,532 INFO - Employee Details page viewed by : conddas
2018-12-03 15:04:27,525 INFO - Employee Details page viewed by : conddas
2018-12-03 15:04:54,428 INFO - Logged out by user..
2018-12-03 15:04:54,428 INFO - -------------------------------------------------------------------------------------
2018-12-03 15:15:21,831 INFO - -------------------------------------------------------------------------------------
2018-12-03 15:15:21,831 INFO - Logged in by: conravh
2018-12-03 15:15:48,800 INFO - Employee Details page viewed by : conravh
2018-12-03 16:23:06,976 INFO - Logged out by user..
2018-12-03 16:23:06,976 INFO - -------------------------------------------------------------------------------------
2018-12-03 16:23:25,909 INFO - -------------------------------------------------------------------------------------
2018-12-03 16:23:25,909 INFO - Logged in by: conplo
2018-12-03 16:23:35,072 INFO - Leave applied for conplo on 11/12/2018 by: conplo
2018-12-03 16:43:42,785 ERROR - Login attempt failed
2018-12-03 16:43:42,785 ERROR - Failed to login for conank
2018-12-03 16:45:42,294 INFO - -------------------------------------------------------------------------------------
2018-12-03 16:45:42,294 INFO - Logged in by: conank
2018-12-03 16:45:53,955 INFO - Logged out by user..
2018-12-03 16:45:53,955 INFO - -------------------------------------------------------------------------------------
2018-12-04 10:06:49,925 INFO - -------------------------------------------------------------------------------------
2018-12-04 10:06:49,925 INFO - Logged in by: conmkum
2018-12-04 10:07:35,977 ERROR - Login attempt failed
2018-12-04 10:07:35,977 ERROR - Failed to login for conmkum
2018-12-04 10:08:18,969 ERROR - Login attempt failed
2018-12-04 10:08:18,969 ERROR - Failed to login for conmkum
2018-12-04 10:10:29,711 INFO - -------------------------------------------------------------------------------------
2018-12-04 10:10:29,711 INFO - Logged in by: conmkum
2018-12-04 10:10:59,006 INFO - Employee Details page viewed by : conmkum
2018-12-04 10:11:10,330 INFO - Leave applied for conmkum on 03/12/2018 by: conmkum
2018-12-04 10:47:10,554 INFO - -------------------------------------------------------------------------------------
2018-12-04 10:47:10,554 INFO - Logged in by: conddas
2018-12-04 10:47:18,018 INFO - Employee Details page viewed by : conddas
2018-12-04 14:55:52,676 INFO - -------------------------------------------------------------------------------------
2018-12-04 14:55:52,676 INFO - Logged in by: consbr
2018-12-04 14:56:41,300 INFO - -------------------------------------------------------------------------------------
2018-12-04 14:56:41,300 INFO - Logged in by: consbr
2018-12-04 14:57:24,373 ERROR - Login attempt failed
2018-12-04 14:57:24,373 ERROR - Failed to login for consbr
2018-12-04 14:58:07,390 ERROR - Login attempt failed
2018-12-04 14:58:07,390 ERROR - Failed to login for consbr
2018-12-04 14:58:50,403 ERROR - Login attempt failed
2018-12-04 14:58:50,403 ERROR - Failed to login for consbr
2018-12-04 15:02:34,375 INFO - -------------------------------------------------------------------------------------
2018-12-04 15:02:34,375 INFO - Logged in by: consbr
2018-12-04 15:05:44,202 INFO - Leave applied for consbr on 03/12/2018 by: consbr
2018-12-04 15:06:20,829 INFO - Leave applied for consbr on 29/11/2018 by: consbr
2018-12-04 15:06:27,561 INFO - Leave applied for consbr on 30/11/2018 by: consbr
2018-12-07 09:11:57,893 INFO - -------------------------------------------------------------------------------------
2018-12-07 09:11:57,893 INFO - Logged in by: conahn
2018-12-07 09:12:07,745 INFO - Leave applied for conahn on 04/12/2018 by: conahn
2018-12-07 09:12:16,185 INFO - Leave applied for conahn on 06/12/2018 by: conahn
2018-12-07 15:23:14,311 INFO - -------------------------------------------------------------------------------------
2018-12-07 15:23:14,311 INFO - Logged in by: conpnag
2018-12-07 15:23:35,982 INFO - Leave applied for conpnag on 10/12/2018 by: conpnag
2018-12-07 15:27:39,395 INFO - Logged out by user..
2018-12-07 15:27:39,395 INFO - -------------------------------------------------------------------------------------
2018-12-07 17:49:08,260 INFO - -------------------------------------------------------------------------------------
2018-12-07 17:49:08,260 INFO - Logged in by: conaku
2018-12-07 17:50:02,990 INFO - Leave applied for conaku on 14/12/2018 by: conaku
2018-12-07 17:50:11,212 INFO - Leave applied for conaku on 20/12/2018 by: conaku
2018-12-10 10:17:16,205 INFO - -------------------------------------------------------------------------------------
2018-12-10 10:17:16,205 INFO - Logged in by: convmis
2018-12-10 10:17:37,427 INFO - Leave applied for convmis on 24/12/2018 by: convmis
2018-12-10 10:17:57,429 INFO - Leave applied for convmis on 25/12/2018 by: convmis
2018-12-10 10:18:03,139 INFO - Leave applied for convmis on 26/12/2018 by: convmis
2018-12-10 10:18:36,808 INFO - Logged out by user..
2018-12-10 10:18:36,808 INFO - -------------------------------------------------------------------------------------
2018-12-10 14:45:27,796 INFO - -------------------------------------------------------------------------------------
2018-12-10 14:45:27,796 INFO - Logged in by: conspatel
2018-12-10 14:46:16,179 INFO - -------------------------------------------------------------------------------------
2018-12-10 14:46:16,179 INFO - Logged in by: conntuk
2018-12-10 14:47:04,644 INFO - -------------------------------------------------------------------------------------
2018-12-10 14:47:04,644 INFO - Logged in by: conntuk
2018-12-10 14:48:09,670 INFO - -------------------------------------------------------------------------------------
2018-12-10 14:48:09,670 INFO - Logged in by: contmg
2018-12-10 14:49:13,911 INFO - -------------------------------------------------------------------------------------
2018-12-10 14:49:13,911 INFO - Logged in by: consys
2018-12-10 14:49:14,036 INFO - Leave applied for conntuk on 03/12/2018 by: conntuk
2018-12-10 14:49:34,800 INFO - Leave applied for contmg on 03/12/2018 by: contmg
2018-12-10 14:50:51,862 INFO - Leave applied for conspatel on 07/12/2018 by: conspatel
2018-12-10 14:51:44,785 INFO - Logged out by user..
2018-12-10 14:51:44,785 INFO - -------------------------------------------------------------------------------------
2018-12-10 14:52:14,614 INFO - Leave applied for conspatel on 21/12/2018 by: conspatel
2018-12-10 14:52:31,976 INFO - Leave applied for conspatel on 24/12/2018 by: conspatel
2018-12-10 14:52:40,628 INFO - Leave applied for conspatel on 25/12/2018 by: conspatel
2018-12-10 15:29:58,060 INFO - Leave applied for conntuk on 04/12/2018 by: conntuk
2018-12-10 15:35:43,683 INFO - Logged out by user..
2018-12-10 15:35:43,683 INFO - -------------------------------------------------------------------------------------
2018-12-11 11:57:52,343 INFO - -------------------------------------------------------------------------------------
2018-12-11 11:57:52,343 INFO - Logged in by: congeth
2018-12-11 11:58:02,527 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-12-11 11:58:10,886 INFO - Leave applied for conrha on 10/12/2018 by: conrha
2018-12-11 11:58:18,295 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-12-11 11:58:33,450 INFO - Leave applied for conshshi on 10/12/2018 by: conshshi
2018-12-11 11:58:44,709 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-12-11 11:59:03,768 INFO - Leave applied for conddas on 10/12/2018 by: conddas
2018-12-11 11:59:24,201 INFO - Leave applied for conddas on 11/12/2018 by: conddas
2018-12-11 11:59:30,410 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-12-11 12:02:16,106 INFO - Logged out by user..
2018-12-11 12:02:16,106 INFO - -------------------------------------------------------------------------------------
2018-12-11 13:56:45,562 INFO - -------------------------------------------------------------------------------------
2018-12-11 13:56:45,562 INFO - Logged in by: conddas
2018-12-11 14:06:42,906 INFO - Logged out by user..
2018-12-11 14:06:42,906 INFO - -------------------------------------------------------------------------------------
2018-12-11 14:33:35,487 INFO - -------------------------------------------------------------------------------------
2018-12-11 14:33:35,487 INFO - Logged in by: congeth
2018-12-11 14:33:42,538 INFO - -------------------------------------------------------------------------------------
2018-12-11 14:33:42,538 INFO - Logged in by: congeth
2018-12-11 14:33:49,161 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-12-11 14:33:50,931 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-12-12 09:58:46,568 ERROR - Login attempt failed
2018-12-12 09:58:46,568 ERROR - Failed to login for consso
2018-12-12 09:59:09,175 INFO - -------------------------------------------------------------------------------------
2018-12-12 09:59:09,175 INFO - Logged in by: consso
2018-12-12 09:59:31,039 INFO - Leave applied for consso on 14/12/2018 by: consso
2018-12-12 17:26:24,361 INFO - -------------------------------------------------------------------------------------
2018-12-12 17:26:24,361 INFO - Logged in by: conmgur
2018-12-12 17:27:27,672 INFO - -------------------------------------------------------------------------------------
2018-12-12 17:27:27,672 INFO - Logged in by: conrank
2018-12-12 17:28:12,570 INFO - Leave applied for conrank on 11/12/2018 by: conrank
2018-12-12 17:28:36,215 INFO - Leave applied for conrank on 10/12/2018 by: conrank
2018-12-12 17:28:49,762 INFO - Leave applied for conrank on 27/12/2018 by: conrank
2018-12-12 17:28:56,725 INFO - Leave applied for conrank on 28/12/2018 by: conrank
2018-12-12 17:29:03,294 INFO - Leave applied for conrank on 31/12/2018 by: conrank
2018-12-12 17:30:50,508 INFO - Employee Details page viewed by : conrank
2018-12-12 23:04:16,844 INFO - -------------------------------------------------------------------------------------
2018-12-12 23:04:16,844 INFO - Logged in by: conprh
2018-12-12 23:04:57,921 INFO - Leave applied for conprh on 05/12/2018 by: conprh
2018-12-12 23:05:06,483 INFO - Leave applied for conprh on 06/12/2018 by: conprh
2018-12-12 23:05:13,216 INFO - Leave applied for conprh on 24/12/2018 by: conprh
2018-12-12 23:05:20,704 INFO - Leave applied for conprh on 31/12/2018 by: conprh
2018-12-12 23:05:37,147 INFO - Logged out by user..
2018-12-12 23:05:37,147 INFO - -------------------------------------------------------------------------------------
2018-12-14 09:24:44,117 INFO - -------------------------------------------------------------------------------------
2018-12-14 09:24:44,117 INFO - Logged in by: conank
2018-12-14 09:26:57,297 INFO - Leave applied for conank on 24/12/2018 by: conank
2018-12-14 09:27:08,444 INFO - Leave applied for conank on 25/12/2018 by: conank
2018-12-14 09:27:21,001 INFO - Logged out by user..
2018-12-14 09:27:21,001 INFO - -------------------------------------------------------------------------------------
2018-12-14 11:40:19,162 INFO - -------------------------------------------------------------------------------------
2018-12-14 11:40:19,162 INFO - Logged in by: conravh
2018-12-14 11:40:23,973 INFO - -------------------------------------------------------------------------------------
2018-12-14 11:40:23,973 INFO - Logged in by: conravh
2018-12-14 11:40:49,471 INFO - Leave applied for conravh on 26/12/2018 by: conravh
2018-12-14 11:40:57,624 INFO - Leave applied for conravh on 27/12/2018 by: conravh
2018-12-14 11:41:03,946 INFO - Leave applied for conravh on 28/12/2018 by: conravh
2018-12-14 11:41:10,630 INFO - Leave applied for conravh on 31/12/2018 by: conravh
2018-12-14 11:41:25,858 INFO - Employee Details page viewed by : conravh
2018-12-14 11:43:10,846 INFO - -------------------------------------------------------------------------------------
2018-12-14 11:43:10,846 INFO - Logged in by: conddas
2018-12-14 11:43:25,759 INFO - Logged out by user..
2018-12-14 11:43:25,759 INFO - -------------------------------------------------------------------------------------
2018-12-14 11:44:12,081 ERROR - Login attempt failed
2018-12-14 11:44:12,081 ERROR - Failed to login for conrak
2018-12-14 11:44:25,457 INFO - -------------------------------------------------------------------------------------
2018-12-14 11:44:25,457 INFO - Logged in by: conrak
2018-12-14 11:44:39,296 INFO - Leave applied for conrak on 10/12/2018 by: conrak
2018-12-14 11:44:45,719 INFO - Leave applied for conrak on 11/12/2018 by: conrak
2018-12-14 11:44:53,977 INFO - Logged out by user..
2018-12-14 11:44:53,977 INFO - -------------------------------------------------------------------------------------
2018-12-14 11:45:09,193 INFO - Logged out by user..
2018-12-14 11:45:09,193 INFO - -------------------------------------------------------------------------------------
2018-12-14 12:00:13,001 INFO - -------------------------------------------------------------------------------------
2018-12-14 12:00:13,001 INFO - Logged in by: conrank
2018-12-14 12:00:19,729 INFO - -------------------------------------------------------------------------------------
2018-12-14 12:00:19,729 INFO - Logged in by: conrank
2018-12-14 12:00:28,018 INFO - -------------------------------------------------------------------------------------
2018-12-14 12:00:28,018 INFO - Logged in by: conrank
2018-12-14 12:01:02,411 INFO - Employee Details page viewed by : conrank
2018-12-14 17:43:42,834 INFO - -------------------------------------------------------------------------------------
2018-12-14 17:43:42,834 INFO - Logged in by: connkh
2018-12-14 17:44:12,745 INFO - Leave applied for connkh on 31/12/2018 by: connkh
2018-12-14 17:55:04,446 INFO - -------------------------------------------------------------------------------------
2018-12-14 17:55:04,446 INFO - Logged in by: conpkr
2018-12-14 17:55:48,426 INFO - Employee Details page viewed by : conpkr
2018-12-14 18:07:07,510 INFO - -------------------------------------------------------------------------------------
2018-12-14 18:07:07,510 INFO - Logged in by: conddas
2018-12-14 19:47:56,436 INFO - -------------------------------------------------------------------------------------
2018-12-14 19:47:56,436 INFO - Logged in by: conasub
2018-12-14 19:48:07,476 INFO - Employee Details page viewed by : conasub
2018-12-14 19:48:26,836 INFO - Employee Details page viewed by : conasub
2018-12-14 20:02:46,980 INFO - Logged out by user..
2018-12-14 20:02:46,980 INFO - -------------------------------------------------------------------------------------
2018-12-17 09:33:51,243 INFO - Logged out by user..
2018-12-17 09:33:51,243 INFO - -------------------------------------------------------------------------------------
2018-12-17 09:34:29,635 INFO - -------------------------------------------------------------------------------------
2018-12-17 09:34:29,635 INFO - Logged in by: conddas
2018-12-17 09:36:06,928 INFO - -------------------------------------------------------------------------------------
2018-12-17 09:36:06,928 INFO - Logged in by: conddas
2018-12-17 09:36:19,187 INFO - Employee Details page viewed by : conddas
2018-12-17 09:36:46,038 INFO - Logged out by user..
2018-12-17 09:36:46,038 INFO - -------------------------------------------------------------------------------------
2018-12-17 09:51:03,788 INFO - -------------------------------------------------------------------------------------
2018-12-17 09:51:03,788 INFO - Logged in by: convmis
2018-12-17 09:51:10,156 INFO - -------------------------------------------------------------------------------------
2018-12-17 09:51:10,156 INFO - Logged in by: conshal
2018-12-17 09:51:21,381 INFO - Leave applied for conshal on 31/12/2018 by: conshal
2018-12-17 10:16:30,783 INFO - Employee Details page viewed by : convmis
2018-12-17 10:17:01,021 INFO - Leave applied for convmis on 27/12/2018 by: convmis
2018-12-17 10:17:11,922 INFO - Logged out by user..
2018-12-17 10:17:11,922 INFO - -------------------------------------------------------------------------------------
2018-12-17 10:26:25,601 INFO - -------------------------------------------------------------------------------------
2018-12-17 10:26:25,601 INFO - Logged in by: conravh
2018-12-17 10:57:23,196 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-12-17 10:57:27,483 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-12-17 10:57:33,786 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-12-17 10:57:35,542 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-12-17 10:57:47,041 INFO - Leave applied for conrhi on 28/12/2018 by: conrhi
2018-12-17 10:57:54,196 INFO - Leave applied for conrhi on 31/12/2018 by: conrhi
2018-12-17 11:22:54,264 INFO - Logged out by user..
2018-12-17 11:22:54,264 INFO - -------------------------------------------------------------------------------------
2018-12-17 11:45:16,136 INFO - -------------------------------------------------------------------------------------
2018-12-17 11:45:16,136 INFO - Logged in by: conpkr
2018-12-17 11:58:48,855 INFO - -------------------------------------------------------------------------------------
2018-12-17 11:58:48,870 INFO - Logged in by: conddas
2018-12-17 12:02:43,537 ERROR - Login attempt failed
2018-12-17 12:02:43,537 ERROR - Failed to login for conmash
2018-12-17 12:03:08,156 ERROR - Login attempt failed
2018-12-17 12:03:08,156 ERROR - Failed to login for conmash
2018-12-17 12:03:36,575 INFO - -------------------------------------------------------------------------------------
2018-12-17 12:03:36,575 INFO - Logged in by: conddas
2018-12-17 12:03:43,391 INFO - Employee Details page viewed by : conddas
2018-12-17 12:04:10,733 INFO - Logged out by user..
2018-12-17 12:04:10,733 INFO - -------------------------------------------------------------------------------------
2018-12-17 12:04:24,472 ERROR - Login attempt failed
2018-12-17 12:04:24,472 ERROR - Failed to login for conamsh
2018-12-17 12:04:38,866 ERROR - Login attempt failed
2018-12-17 12:04:38,866 ERROR - Failed to login for conamsh
2018-12-17 12:08:54,369 ERROR - Login attempt failed
2018-12-17 12:08:54,369 ERROR - Failed to login for conamsh
2018-12-17 12:09:11,976 ERROR - Login attempt failed
2018-12-17 12:09:11,976 ERROR - Failed to login for conamsh
2018-12-17 12:09:19,022 ERROR - Login attempt failed
2018-12-17 12:09:19,022 ERROR - Failed to login for conamsh
2018-12-17 12:09:22,808 ERROR - Login attempt failed
2018-12-17 12:09:22,808 ERROR - Failed to login for conamsh
2018-12-17 12:09:41,697 ERROR - Login attempt failed
2018-12-17 12:09:41,697 ERROR - Failed to login for conamsh
2018-12-17 12:10:02,454 ERROR - Login attempt failed
2018-12-17 12:10:02,454 ERROR - Failed to login for conamsh
2018-12-17 12:21:06,310 INFO - Employee Details page viewed by : conddas
2018-12-17 12:21:56,641 INFO - Logged out by user..
2018-12-17 12:21:56,641 INFO - -------------------------------------------------------------------------------------
2018-12-17 12:30:31,236 INFO - -------------------------------------------------------------------------------------
2018-12-17 12:30:31,236 INFO - Logged in by: conpkr
2018-12-17 13:48:55,726 INFO - -------------------------------------------------------------------------------------
2018-12-17 13:48:55,726 INFO - Logged in by: congeth
2018-12-17 13:49:56,936 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-12-17 13:50:11,215 INFO - Leave applied for conavk on 13/12/2018 by: conavk
2018-12-17 13:50:16,794 INFO - Leave applied for conavk on 14/12/2018 by: conavk
2018-12-17 13:53:13,946 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-12-17 13:57:51,638 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-12-17 16:56:28,226 INFO - Logged out by user..
2018-12-17 16:56:28,226 INFO - -------------------------------------------------------------------------------------
2018-12-17 17:11:34,596 INFO - -------------------------------------------------------------------------------------
2018-12-17 17:11:34,596 INFO - Logged in by: conravh
2018-12-17 17:11:41,501 INFO - -------------------------------------------------------------------------------------
2018-12-17 17:11:41,501 INFO - Logged in by: conravh
2018-12-17 17:11:48,159 INFO - -------------------------------------------------------------------------------------
2018-12-17 17:11:48,159 INFO - Logged in by: conravh
2018-12-17 17:13:23,338 INFO - -------------------------------------------------------------------------------------
2018-12-17 17:13:23,338 INFO - Logged in by: consbr
2018-12-17 17:13:45,317 INFO - Leave applied for consbr on 14/12/2018 by: consbr
2018-12-17 17:13:53,875 INFO - Leave applied for consbr on 31/12/2018 by: consbr
2018-12-17 17:25:13,782 INFO - -------------------------------------------------------------------------------------
2018-12-17 17:25:13,782 INFO - Logged in by: consys
2018-12-17 17:25:20,363 INFO - -------------------------------------------------------------------------------------
2018-12-17 17:25:20,363 INFO - Logged in by: consys
2018-12-17 17:27:06,396 INFO - Leave applied for consys on 20/12/2018 by: consys
2018-12-17 17:27:14,669 INFO - Logged out by user..
2018-12-17 17:27:14,669 INFO - -------------------------------------------------------------------------------------
2018-12-17 17:32:31,120 INFO - Logged out by user..
2018-12-17 17:32:31,120 INFO - -------------------------------------------------------------------------------------
2018-12-18 09:48:48,816 INFO - -------------------------------------------------------------------------------------
2018-12-18 09:48:48,816 INFO - Logged in by: conplo
2018-12-18 09:48:55,426 INFO - -------------------------------------------------------------------------------------
2018-12-18 09:48:55,426 INFO - Logged in by: conplo
2018-12-18 09:49:18,617 INFO - Logged out by user..
2018-12-18 09:49:18,617 INFO - -------------------------------------------------------------------------------------
2018-12-19 09:33:12,265 INFO - -------------------------------------------------------------------------------------
2018-12-19 09:33:12,265 INFO - Logged in by: conplo
2018-12-19 09:33:33,641 INFO - Leave applied for conplo on 17/12/2018 by: conplo
2018-12-19 12:41:40,347 INFO - -------------------------------------------------------------------------------------
2018-12-19 12:41:40,347 INFO - Logged in by: conaku
2018-12-19 12:41:53,068 INFO - Leave applied for conaku on 19/12/2018 by: conaku
2018-12-19 12:42:04,087 INFO - Logged out by user..
2018-12-19 12:42:04,087 INFO - -------------------------------------------------------------------------------------
2018-12-19 13:35:31,654 INFO - -------------------------------------------------------------------------------------
2018-12-19 13:35:31,654 INFO - Logged in by: conddas
2018-12-19 13:35:55,711 INFO - Employee Details page viewed by : conddas
2018-12-19 13:36:51,894 INFO - Logged out by user..
2018-12-19 13:36:51,894 INFO - -------------------------------------------------------------------------------------
2018-12-19 14:00:54,618 INFO - -------------------------------------------------------------------------------------
2018-12-19 14:00:54,618 INFO - Logged in by: conravh
2018-12-19 14:00:58,932 INFO - -------------------------------------------------------------------------------------
2018-12-19 14:00:58,932 INFO - Logged in by: conravh
2018-12-19 17:35:01,589 INFO - Logged out by user..
2018-12-19 17:35:01,589 INFO - -------------------------------------------------------------------------------------
2018-12-20 09:08:46,009 INFO - -------------------------------------------------------------------------------------
2018-12-20 09:08:46,009 INFO - Logged in by: conssud
2018-12-20 09:09:25,586 INFO - Leave applied for conssud on 19/12/2018 by: conssud
2018-12-20 09:09:45,962 INFO - Logged out by user..
2018-12-20 09:09:45,962 INFO - -------------------------------------------------------------------------------------
2018-12-20 09:16:16,692 INFO - -------------------------------------------------------------------------------------
2018-12-20 09:16:16,692 INFO - Logged in by: convmis
2018-12-20 09:16:23,508 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-12-20 09:16:28,147 INFO - Leave applied for convmoh on 11/12/2018 by: convmoh
2018-12-20 09:16:32,406 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-12-20 09:16:32,422 ERROR - Exception on /showPersonalLeave [GET]
Traceback (most recent call last):
  File "C:\Python\lib\site-packages\flask\app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Python\lib\site-packages\flask\app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Python\lib\site-packages\flask\_compat.py", line 33, in reraise
    raise value
  File "C:\Python\lib\site-packages\flask\app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Python\lib\site-packages\flask\app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "EmployeeProfileUI.py", line 590, in showPersonalLeave
    print(" : Inside data : " + corp_id_org)
TypeError: must be str, not NoneType
2018-12-20 09:16:38,822 INFO - Logged out by user..
2018-12-20 09:16:38,822 INFO - -------------------------------------------------------------------------------------
2018-12-20 09:20:55,839 INFO - -------------------------------------------------------------------------------------
2018-12-20 09:20:55,839 INFO - Logged in by: convmis
2018-12-20 09:21:23,412 INFO - -------------------------------------------------------------------------------------
2018-12-20 09:21:23,412 INFO - Logged in by: convmis
2018-12-20 09:21:52,243 INFO - -------------------------------------------------------------------------------------
2018-12-20 09:21:52,243 INFO - Logged in by: convmis
2018-12-20 09:22:20,793 INFO - -------------------------------------------------------------------------------------
2018-12-20 09:22:20,793 INFO - Logged in by: convmis
2018-12-20 09:23:00,115 INFO - -------------------------------------------------------------------------------------
2018-12-20 09:23:00,115 INFO - Logged in by: convmis
2018-12-20 09:24:12,289 INFO - Employee Details page viewed by : convmis
